packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -538,7 +538,8 @@ const deleteCfnStack = async (account: AWSAccountInfo, accountIndex: number, sta
   try {
     const cfnClient = new aws.CloudFormation(getAWSConfig(account, region));
     await cfnClient.deleteStack({ StackName: stackName, RetainResources: resourceToRetain }).promise();
-    await cfnClient.waitFor('stackDeleteComplete', { StackName: stackName }).promise();
+    // we'll only wait up to 10 minutes before moving on
+    await cfnClient.waitFor('stackDeleteComplete', { StackName: stackName, $waiter: { 'maxAttempts': 20 }}).promise();
```

Comment:
nice !

Reply:
good question; I considered it but did 10 minutes to be safe, this keeps the job as a whole within 10-15 minutes each time